tasks/fr/fr_income.py

```diff
@@ -0,0 +1,197 @@
+                    targets_dict[cols.get(x)] = 'denominator'
+                targets_dict.pop(None, None)
+                universes = universe.split(',')
+                for x in universes:
+                    x = x.strip()
+                    targets_dict[cols.get(x)] = 'universe'
+                targets_dict.pop(None, None)
+                cols[var_code] = OBSColumn(
+                    id=var_code,
+                    type='Numeric',
+                    name=long_name,
+                    description =long_name,
+                    # Ranking of importance, sometimes used to favor certain measures in auto-selection
+                    # Weight of 0 will hide this column from the user.  We generally use between 0 and 10
+                    weight=5,
+                    aggregate='',
+                    # Tags are our way of noting aspects of this measure like its unit, the country
+                    # it's relevant to, and which section(s) of the catalog it should appear in
+                    tags=[france, unittags[var_unit]],
+                    targets= targets_dict
+                )
+                subsections = subsections.split(',')
+                for s in subsections:
+                    s = s.strip()
+                    subsection_tag = subsectiontags[s]
+                    cols[var_code].tags.append(subsection_tag)
+                if var_unit in ('households', 'people', 'tax_consumption_units'):
+                    cols[var_code].aggregate = 'sum'
+
+        for _,col in cols.iteritems():
+            col.tags.append(insee_source)
+        return cols
+
+class FranceIncome(TableTask):
+
+    table_theme = Parameter()
+
+    def version(self):
+        return 2
+
+    def timespan(self):
+        return '2012'
+
+    def requires(self):
+        requirements = {
+            'data': RawIncomeIrisData(table_theme=self.table_theme),
+            'meta': IrisIncomeColumns(table_theme=self.table_theme),
+            'geometa': OutputAreaColumns(),
+        }
+        return requirements
+
+    def columns(self):
+        cols = OrderedDict()
+        cols['IRIS'] = self.input()['geometa']['dcomiris']
+        cols.update(self.input()['meta'])
+        return cols
+
+    def populate(self):
+        session = current_session()
+
+        column_targets = self.columns()
+        colnames = ', '.join(column_targets.keys())
+        colnames_typed = ','.join(['{}::{}'.format(colname, ct.get(session).type)
+                              for colname, ct in column_targets.iteritems()])
+        session.execute('INSERT INTO {output} ({ids}) '
+                        'SELECT {ids_typed} '
+                        'FROM {input} '.format(
+                            ids=colnames,
+                            ids_typed=colnames_typed,
+                            output=self.output().table,
+                            input=self.input()['data'].table
+                        ))
+
+
+class IRISIncomeTables(WrapperTask):
```

Comment:
Do we need a meta wrapper? I think at least to test this part automatically is needed